Core/include/Acts/EventData/TrackProxy.hpp

```diff
@@ -411,7 +411,7 @@ class TrackProxy {
   /// Get the particle hypothesis
   /// @return the particle hypothesis
   ParticleHypothesis particleHypothesis() const {
-    return component<ParticleHypothesis, hashString("particleHypothesis")>();
+    return m_container->container().particleHypothesis_impl(m_index);
```

Comment:
It's not strictly related, but I noticed the particle hypothesis copy was not working in PODIO, because I forgot that it'll have to pack/unpack this type. There should be a setter as well now, I had previously forgotten it in the PODIO backend.

I'm also noticing now that the particle hypo is not part of the backend concept yet. I'll add this.